.github/workflows/build_docs.yaml

```
@@ -7,30 +7,34 @@ on:
           - windows-latest
+        task:
+          - Build
+        include:
+          - task: Deploy
+            os: ubuntu-latest
+            python-version: "3.13"
+            deploy: true
       fail-fast: false
     runs-on: ${{ matrix.os }}
+    # Only run deploy job on push to master
+    if: ${{ !(matrix.deploy == true) || (github.event_name == 'push' && github.ref == 'refs/heads/master') }}
```

actionlint (1.7.4)

context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)